src/main.cc

```diff
@@ -59,7 +63,16 @@ int main(int argc, char *argv[]) {
             use_captured_sym_map = map_type == 2;
             break;
         case 'v':
-            verbosity = atoi(optarg);
+            verb_trim = strtok(optarg, " ");
+            strcpy(verbstr, verb_trim);
+            vopt = strtok(verb_trim, ",");
+            while (vopt != NULL) {
+                if (!strstr(vstr, vopt)) {
+                    verr = true;
```

Comment:
should error be true here as well?

Reply:
error gets set to true if verr is true in line 122.

Reply:
Now line 131